src/corelibs/U2View/src/ov_msa/MSAEditor.cpp

```diff
@@ -755,18 +767,14 @@ void MSAEditor::sl_showCustomSettings() {
     AppContext::getAppSettingsGUI()->showSettingsDialog(ColorSchemaSettingsPageId);
 }
 
-void MSAEditor::sortSequences(bool isByName, const MultipleAlignment::Order &sortOrder) {
+void MSAEditor::sortSequences(const MultipleAlignment::SortType &sortType, const MultipleAlignment::Order &sortOrder) {
```

Comment:
Enums a too simple types to pass them as const references.

Reply:
Fixed